app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragmentInteractor.kt

```diff
@@ -41,6 +41,7 @@ class BookmarkFragmentInteractor(
     }
 
     override fun onSearch() {
+        BookmarksManagement.searchIconTapped.record(NoExtras())
```

Comment:
should this instead live inside the controller?

Reply:
I thought about this as well. But seeing cases such as `onCopyPressed` or `onSharePressed` below which records their metric outside the controller I decided to follow that convention for consistency.

Reply:
kk, no worries then. it can be a follow-up to re-locate them all down the road.